com.unity.render-pipelines.high-definition/Runtime/RenderPipeline/RenderPass/AOV/AOVRequestData.cs

```diff
@@ -200,6 +201,24 @@ List<RTHandle> targets
             }
         }
 
+        internal void OverrideBufferFormatForAOVs(ref GraphicsFormat format, List<RTHandle> aovBuffers)
```

Comment:
Aren't we missing somewhere an update of the documentation? 
also feel the what's new in 11 pages